build.gradle

```diff
@@ -6,9 +6,8 @@ buildscript {
         maven { url "https://maven.fullstory.com" }
     }
     dependencies {
-        classpath "com.fullstory:instrumentation-full:1.3.0"
```

Comment:
You are correct to remove this

Reply:
Thanks for confirming! Not sure how it got there. Oops.